src/components/TabBar/TabBar.css

```diff
@@ -41,11 +41,4 @@
     border-color: var(--primaryLinkColor);
     color: color(var(--colorMuted) lightness(85%));
   }
-
-  @media only screen and (max-width: 1439px) {
-    white-space: nowrap;
```

Comment:
I still prefer the scrolling on mobile, but it's not as bad as I thought. I think we can do better than this, but if you want to go ahead with this merge I won't hold it back.